guidelines/README.md

```diff
@@ -0,0 +1,5 @@
+# Guidelines
+
+The objective of guidelines is to share knowledge between club leaders in a more
```

Comment:
@paked Thanks for the input! I'll close the PR for now and we can reopen it if anyone ever has questions about what to put in the `/guidelines` folder.